yang-models/dell-base-alms.yang

```diff
@@ -22,12 +22,16 @@ module dell-base-alms {
 
   All rights reserved.";
 
+  revision 2018-09-21 {
```

Comment:
We have taken care of most of the pyang errors internally. They should be available soon.
In this case, you should write the revision as:

 revision 2018-09-21 {
    description "Fixes for error from pyang validation and IETF compliance";
    reference "Network Platform Abstraction";
  }

Without the reference, pyang would through IETF compliance errors.